.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,7 +34,7 @@ that might be useful for reviewers.. -->
 
 ## Acknowledgements
 - [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
-
+- [ ] The PR title is descriptive enough
```

Comment:
The decision whether the PR title is descriptive enough or not is on the reviewer. It's still a good idea to have a checkbox for PR authors as well to make them aware of the issue. Suggestion for slight rewording:

```suggestion
- [ ] I have chosen a descriptive PR title (see top of PR template for examples)
```

Reply:
Sounds great!